.github/test/crystal-libs_spec.cr

```diff
@@ -0,0 +1,47 @@
+require "spec"
+require "yaml"
+
+it "yaml" do
+  any = YAML::Any.new({
+    YAML::Any.new("foo") => YAML::Any.new(12_i64)
+  })
+  YAML.parse(any.to_yaml).should eq any
+end
+
+require "xml"
+
+it "xml" do
+  string = XML.build do |xml|
+    xml.element("person", id: 1) do
+      xml.element("firstname") { xml.text "Jane" }
+      xml.element("lastname") { xml.text "Doe" }
+    end
+  end
+
+  document = XML.parse(string)
+  person = document.first_element_child
+  if person
```

Comment:
What if not person? 😃 That would be an even worse error but it's not detected here. This should raise if not person, then the following code can also be un-indented